content/docs/guides/scim/okta.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Configuring SCIM in Okta
+meta_desc: This page describes how to support SCIM 2.0 functionality between Pulumi and Okta.
+menu:
+    userguides:
+        parent: scim
+        weight: 1
+---
+
+This document covers steps to help you configure automatic provisioning/deprovisioning of your users in Pulumi using SCIM 2.0.
+
+Please note that not all SCIM features are supported. For more information, see [Known Limitations]({{< relref "#known-limitations" >}}).
+
+## Prerequisites
+
+* Your organization must already be configured to use [SAML SSO]({{< relref "/docs/guides/saml/okta" >}}) with Pulumi.
+* You must be an admin of your organization.
+* (Optional) It is highly recommended that you have more than one admin for your org.
+
+## Enabling SCIM For Your Okta Org
+
+In order to configure SCIM for your Okta tenant, email support@okta.com requesting them to enable `SCIM_PROVISIONING` for your Okta org.
+
+Once the feature is enabled, navigate to the Pulumi SAML app configured in your Okta org from the **Classic UI**. Under the **General** tab, in the **App Settings** dialog box, locate the **Provisioning** section and select **SCIM**. Click **Save** to save your changes and close the **App Settings** dialog box.
```

Comment:
This Okta doc has some decent screenshots (that they, not us, have the burden to update ;)) - would it be worth a link?

https://support.okta.com/help/s/article/SCIM-Provisioning-Enabled-in-Custom-App?language=en_US

Reply:
Agreed. Added link as part of https://github.com/pulumi/docs/pull/5139/commits/3b7f1b843826a1455935c066cda6e5bfbf01fb11.